lib/kernel-manager.js

```diff
@@ -101,11 +100,7 @@ class KernelManager {
         break;
     }
 
-    if (projectPath != null) {
-      kernelStartDir = projectPath;
-    } else {
-      kernelStartDir = currentPath;
-    }
+    const kernelStartDir = projectPath != null ? projectPath : currentPath;
```

Comment:
:) cool! unlearning this kind of shortening stuff w/ `go` atm but looks reasonable :)

@n-riesco this is not a string but the `null`/"undefined" keyword?